apps/admin-ui/src/authentication/AuthenticationSection.tsx

```diff
@@ -81,7 +82,12 @@ export default function AuthenticationSection() {
     );
     const flows = await flowsRequest.json();
 
-    return sortBy(flows as AuthenticationType[], (flow) => flow.usedBy?.type);
+    return sortBy(
+      flows
+        ? localeSort(flows as AuthenticationType[], mapByKey("alias"))
+        : ([] as AuthenticationType[]),
+      (flow) => flow.usedBy?.type
+    );
```

Comment:
Couple of points here.

1. You can handle `flows` possibly being `undefined` with an early-return (removes the need to re-check it later on).
2. You can pass `AuthenticationType` as a generic to `localeSort` (removes the need to cast it twice).

E.g.:

```suggestion
    if (!flows) {
      return [];
    }

    return sortBy(
      localeSort<AuthenticationType>(flows, mapByKey("alias")),
      (flow) => flow.usedBy?.type
    );
```